source/includes/migration-upgrade-recommendation.rst

```diff
@@ -0,0 +1,3 @@
+For best performance, upgrade your source and destination clusters to
+the most recent server patch release prior to migration. For more
+information, see :ref:`MongoDB Patch Releases <versions-patch-release>`.
```

Comment:
Maybe this page might be slightly better to link to:

https://www.mongodb.com/docs/manual/tutorial/upgrade-revision/

Reply:
Done! Thanks for this suggestion.